release/models/bfd/openconfig-bfd.yang

```diff
@@ -157,35 +164,41 @@ module openconfig-bfd {
           "The BFD echo function failed - echo packets have not been
           received for the required period of time.";
       }
-      enum FORWARDING_RESET {
+      enum NEIGHBOR_DOWN {
```

Comment:
To keep this as a non-breaking change, the original ordering of the enums must be preserved.  You can only add enums to the list.

Reply:
I realize the enum value won't match the value in a packet on the wire, but that is not required.